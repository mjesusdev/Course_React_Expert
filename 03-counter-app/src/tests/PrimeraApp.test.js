import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import PrimeraApp from '../PrimeraApp';

describe('Pruebas en <PrimeraApp />', () => {
  
/*     test('should show message Hola soy Goku', () => {
        const saludo = 'Hola, soy Goku';

        const { getByText } = render( <PrimeraApp saludo = { saludo }/> );

        expect( getByText(saludo) ).toBeInTheDocument();
    }) */

    test('should shows <PrimeraApp />', () => {
        const saludo = 'Hola, Soy Goku';
        const wrapper = shallow( <PrimeraApp saludo = { saludo } /> )

        expect( wrapper ).toMatchSnapshot()
    });

/*     test('should shows subtitle send for props', () => {
        const saludo = 'Hola, Soy Goku';
        const subtitle = 'Soy un subt√≠tulo';
        const wrapper = shallow(
            <PrimeraApp
                saludo = { saludo }
                subtitulo = { subtitle }
            />
        )

        const textParragraph = wrapper.find('p').text();

        expect( textParragraph ).toBe( subtitle );
    }); */
})