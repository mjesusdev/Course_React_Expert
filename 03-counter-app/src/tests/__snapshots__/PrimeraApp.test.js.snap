// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp /> should shows <PrimeraApp /> 1`] = `
<Fragment>
  <h1>
     
    Hola, Soy Goku
     !
  </h1>
  <p>
     
    Soy un subtitulo
     
  </p>
</Fragment>
`;
